snippets/strings.yml

```diff
@@ -1,9 +1,34 @@
 bytes_previous_versions_link: All previously released versions of this data are
-  available at <a
-  href="https://www.nyc.gov/site/planning/data-maps/open-data/bytes-archive.page?sorts%5Byear%5D=0">BYTES
-  of the BIG APPLE - Archive</a>.
+  available at <a href="https://www.nyc.gov/content/planning/pages/resources#datasets">BYTES of the BIG APPLE - Archive</a>.
 streets_map_app_link: This dataset is featured on the Department of City Planning's <a
   href="https://streets.planning.nyc.gov/">Street Map application</a>.
 bytes_zip_prefix_url: https://s-media.nyc.gov/agencies/dcp/assets/files/zip/data-tools/bytes
 bytes_pdf_prefix_url: https://s-media.nyc.gov/agencies/dcp/assets/files/pdf/data-tools/bytes
 bytes_xl_prefix_url: https://s-media.nyc.gov/agencies/dcp/assets/files/excel/data-tools/bytes
+
+## BYTES urls
```

Comment:
The format for these links leaves something to be desired. But DRY'ing them up by e.g. snippet'ing out the `https://www.nyc.gov/content/planning/pages/resources/` actually harmed readability, I thought. 

Sometimes dumber is better.